bundles/org.openhab.core.io.monitor/bnd.bnd

```diff
@@ -6,4 +6,6 @@ Import-Package: \
  org.osgi.service.*,\
  org.slf4j.*,\
  com.google.gson.*;version="[2.8,3)"
-Export-Package: io.micrometer.core.*
+Export-Package: io.micrometer.core.*;-split-package:=merge-first,\
+ org.HdrHistogram.*;-split-package:=merge-first,\
```

Comment:
Are those libraries included in the bundle jar?

Reply:
Yes.